openshift-install.md

```diff
@@ -9,10 +9,29 @@ for your OpenShift installation to be able to fetch images from this registry.
 
 First, complete the instructions
 [here](https://access.redhat.com/terms-based-registry/#/token/openshift3-test-cluster/docker-config). Unfortunatelly,
```

Comment:
Can you please change this endpoint to the correct one? (the one that has the yaml file you are referring to, which is this https://access.redhat.com/terms-based-registry/#/token/openshift3-test-cluster/openshift-secret).

Also, the name for the file to download is `openshift3-test-cluster-secret.yaml`